data/jibauni/jibauni 0 kanlanmo transport.txt

```diff
@@ -0,0 +1,665 @@
+			choice
+				`	"Why not? There's nothing else to do."`
+					accept
+				`	"In the sake of establishing a good relationship, sure!"`
+					accept
+				`	"Of course, but unfortunately I don't have space right now. I can come back later though."`
+			`	"Okay," says Kanlanmo. "Chances are, I still won't have found transport, so I will be waiting here next time you come."`
+				decline
+			label notseen
+			`	"Okay. In that case, I have another request for you. Can you take me to <planet> in the <system> system? I will mark it on your map, and explain more once we get there. Oh yes, and my name is Kanlanmo. Kanlanmo of the Jibauni."`
+			choice
+				`	"Of course! You can get on the <ship> right away!"`
+					accept
+				`	"Why?"`
+					goto whychoice
+				`	"I'm sorry, I can't take you now. Later maybe though."`
+					decline
+			label wormholeentry
+			`	"Oh, we have seen it," says the alien. "But it does not let our ships though. The reason for this is unclear. but to matters of more importance. We have seen an alien ship in pink markings pass through. Have you seen it, or one like it?"`
+				goto pinkship
+			label alphas
+			`	You begin the story, telling everything you know about how the Alphas tried to enslave human space.`
+			`	At the end, he says, "Wow. That is a long story. And I take it you are not on their side?"`
+			choice
+				`	"No, of course not."`
+					goto alphasoffer
+				`	"Yes, I have been friendly with them for years."`
+					goto friendlyalpha
+			label alphasoffer
+			`	"Good. Well, that just leaves one more matter to be wrapped up. Can you take me to <planet> in the <system> system? Kanlanmo of the Jibauni will be grateful."`
+			choice
+				`	"Of course!"`
+					accept
+				`	"Why?"`
+					goto whychoice
+				`	"Not yet, I have something I need to do first. But I'll be back shortly!"`
+					decline
+			label rude
+			`	The alien announces, though to what you cannot see, "The stranger has been ignominiously impolite." It turns to you. "What do you say, 'weirdo'? Will you apologize? Or not, and see what happens?"`
+			choice
+				`	"I apologize, and did not mean to insult you."`
+					goto rudeentry
+				`	"Why apologize to a bunch of dinky T-Rex's?"`
+					goto deathexplode
+			label deathexplode
+			`	A cluster of anti-ship weaponry is fired from all sides, immediately exploding the <ship> and everything inside it. It is sad your life had to end in such a way.`
+				die
+			label deathshot
+			`	You pull your sidearm from its holster, but before you can fire it, hidden anti-ship weaponry trains fire on your ship. Caught in crossfire, the <ship> immediately explodes, blowing you and everything in it sky high.`
+				die
+			label friendlyalpha
+			action
+				"reputation: Jibauni" = -1000
+				"reputation: Jibauni (Civilian)" = -1000
+				"reputation: Jibauni (Fueler)" = -1000
+				"reputation: Jibauni (Rebel)" = -1000
```

Comment:
Because, while the Jibauni and Rebel Jibauni are on fighting terms, they loathe the Alphas infinitely more than they could ever feel emotion for each other.

The Jibauni, hearing what First Last has to say, are going to send a truce ship to give the message.